src/promptflow/promptflow/executor/flow_validator.py

```diff
@@ -50,9 +51,10 @@ def _ensure_nodes_order(flow: Flow):
                 # Figure out the nodes names with circular dependency problem alphabetically
                 remaining_nodes = sorted(list(set(dependencies.keys()) - picked))
                 raise NodeCircularDependency(
-                    message=f"Node circular dependency has been detected among the nodes in your flow. "
-                    f"Kindly review the reference relationships for the nodes {remaining_nodes} "
-                    f"and resolve the circular reference issue in the flow."
+                    message_format="Flow is defined incorrectly. Node circular dependency has been detected among the "
+                    "nodes in your flow. Kindly review the reference relationships for the nodes "
+                    "{remaining_nodes} and resolve the circular reference issue in the flow.",
```

Comment:
Let's use this style for multi-line error message formats:
![image](https://github.com/microsoft/promptflow/assets/113209/0ce9cc4a-4070-4ef3-b907-0fad38ce9f14)